cobalt/xhr/xml_http_request.cc

```diff
@@ -618,14 +619,18 @@ void XMLHttpRequestImpl::SendFallback(
   DCHECK_CALLED_ON_VALID_THREAD(thread_checker_);
   // Step 1
   if (state_ != XMLHttpRequest::kOpened) {
-    web::DOMException::Raise(web::DOMException::kInvalidStateErr,
-                             exception_state);
+    if (exception_state) {
```

Comment:
I think what's going wrong is the passing of `exception_state` to the `PostTask` above, because when that task eventually gets processed, that may have become an invalid pointer.

Reply:
Can we somehow repro this error and have a test for it to prove the fix works?